src/main/java/com/prgms/voucher/voucherproject/domain/FixedAmountVoucher.java

```diff
@@ -0,0 +1,30 @@
+package com.prgms.voucher.voucherproject.domain;
+
+import java.util.UUID;
+
+public class FixedAmountVoucher implements Voucher{
+    private final UUID voucherId;
+    private final long amount;
+
+    public FixedAmountVoucher(UUID voucherId, long amount) {
+        this.amount = amount;
+        this.voucherId = voucherId;
+    }
+
+
+    @Override
+    public UUID getVoucherId() {
+        return voucherId;
+    }
+
+    public long discount(long beforeDiscount) {
+        return beforeDiscount - amount;
+    }
+
+    @Override
+    public String toString() {
+        return "| UUID:" + getVoucherId() + " | VoucherType: FixedVoucher | amount:" + amount +" |";
```

Comment:
띄어쓰기가 빠져있어서 코드 제안 드려보았어요.
```suggestion
        return "| UUID:" + getVoucherId() + "  | VoucherType: PercentVoucher | percent:" + percent + " |";
```